code/modules/station_goals/bluespace_tap.dm

```diff
@@ -197,6 +197,7 @@
 	density = TRUE
 	interact_offline = TRUE
 	luminosity = 1
+	anchored = TRUE
```

Comment:
`machinery/power` is anchored by default, does this do anything or is the fix solely the `connect_to_network()`?

Reply:
thanks thats removed